test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -492,29 +500,47 @@ function source(::EntrDetr{up_ρa{i}}, atmos, args) where {i}
     return E_dyn[i] - Δ_dyn[i]
 end
 
-function source(::EntrDetr{up_ρaw{i}}, atmos, args) where {i}
+function source(
+    ::EntrDetr{up_ρaw{i}},
+    atmos::AtmosModel{FT},
+    args,
+) where {i, FT}
```

Comment:
```suggestion
function source(::EntrDetr{up_ρaw{i}}, atmos, args) where {i}
    FT = eltype(atmos)
```
Then, let's just define `Base.eltype(::AtmosModel{FT}) where {FT} = FT` in `src/Atmos/Model/Atmos.jl` (somewhere after `AtmosModel` is defined).

Reply:
We can use the same pattern below